MycroftAI/selene-backend
# Change Account API to handle an email change request

## Description
When a user requests an email address change, sends an email to old and new addresses.  New address contains a verification link.  Email address change is not applied until new address is verified.

## How to test
This PR must be tested in tandem with a PR in the selene-ui repository that adds the change email button.  Click the button and verify email address.  Then login with new email address.

## Contributor license agreement signed?
CLA [X] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)
